### DISCUSSION TITLE
What setting can cause emojis to trigger a misalignment of text in the history list?

### DISCUSSION THREAD

**Discussion by Aster89** [qutebrowser/qutebrowser#7767] [Q&A]
I have not created a bug report because the issue doesn't happen if I start with `--temp-basedir`.

But here it is:
![qutebrowserHistoryWithEmojiHight](https://github.com/qutebrowser/qutebrowser/assets/20521900/5b39b1d0-f0b0-4b1d-8313-28f8e05fe971)

Any clue what setting I clould look at?

**Comment by The-Compiler** [comment]
Sorry, this got lost :)

The font settings might be the culprit. Somewhat related:

- #2112

**Reply by Aster89** [reply]
No problem, thanks for the info.
